0000DCA632B3FD5F.json

```diff
@@ -1,6 +1,6 @@
 {
         "gateway_conf": {
-                "gateway_ID": " 0000DCA632B3FD5",
+                "gateway_ID": " 0000DCA632B3FD5F",
```

Comment:
The space after the double quote will break the parsing of the file. 
But, I ask again: are you sure this is your MAC address? Are you installing this on a raspberry pi or another board?

Reply:
Sorry. I'm not very familiar with Github. Yes I'm installing on a Rasberry Pi. I alread have a gateway running for a few years, but this is the first time I do it with remote config. MAC address dc:a6:32:b3:fd:5e for eth0 and dc:a6:32:b3:fd:5f for wlan0.

Reply:
But if you are on a raspberry pi, I assume you're using the install on [this wiki](https://github.com/ttn-zh/ic880a-gateway/wiki) and in that case, the installer will tell you which MAC address you need to use, and it's one that in the overwhelming majority of cases starts with `B827EBFFEE`